src/EA.Weee.Web/Areas/Admin/Views/Aatf/Details.cshtml

```diff
@@ -176,11 +176,85 @@
                     <th class="govuk-table__header" scope="col">Email address</th>
                     <td class="govuk-table__cell">@Model.Organisation.BusinessAddress.Email</td>
                 </tr>
+                @if (Model.AssociatedAatfs.Any() || Model.AssociatedAEs.Any() || Model.AssociatedSchemes.Any())
```

Comment:
Nicer to create bool property on Model for example AnyRelatedEntities that would return true if any of these properties have values, then can unit test :)

Reply:
Added :)